### PR TITLE
Scout API Manager configuration

### PR DESCRIPTION
**Pull request by hprobotic**
- Add Scout API Manager
- Add API URI configuration
- Add List all issue view and controller

### CODE REVIEW

**Comment by hprobotic** [comment]
@rockerhieu anh xem giup em voi anh nhe.

File: Scout/Models/Issue.swift [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  var category: Category!
+  var votesCount = 0
+  var photos: [Photo]?
+  var location: Location?
+  var createdAt: NSDate!
+  var modifiedAt: NSDate!
+  var isAnonymous: Bool?
+  
+  static let sharedDateFormatter = Issue.dateFromatter()
+
```
Comment (rockerhieu) [inline Scout/Models/Issue.swift]: Why do we have so many empty lines here?

File: Scout/ResponseJSONObjectSerializable.swift [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+//  Scout
+//
+//  Created by Johnny Pham on 4/15/16.
+//  Copyright © 2016 Team Nato. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+public protocol ResponseJSONObjectSerializable {
```
Comment (rockerhieu) [inline Scout/ResponseJSONObjectSerializable.swift:12]: What is this class used for?

Comment (hprobotic) [inline Scout/ResponseJSONObjectSerializable.swift:12]: That extension let us handle a response from Alamofire that returned an object as JSON and
create a Swift object (as long as the Swift class had the right kind of initializer, as per the
ResponseJSONObjectSerializable protocol). Now we need something similar but for a whole array of objects: We want to take a JSON array and turn it into an array of Swift objects. We’ll keep our protocol so we need to add to our project. Add a new ResponseJSONObjectSerializable.swift file and add the protocol to it.

File: Scout/Models/Issue.swift (hunk 1 above)
Comment (hprobotic) [inline Scout/Models/Issue.swift]: I'll remove all empty line. Thanks 👍

**Pull request merged**